### PR TITLE
When migrating from Gitlab map Approvals to approving Reviews

### LINKED ISSUE #11137: [Feature] add migration support for Reviews to gitlab migration
gitlab migration was introduced in #9084 but without **GetReviews** functionalety

after #11136 got merged ... check if the client now support reviews as described in:
https://github.com/go-gitea/gitea/pull/9084#issuecomment-610914976

ref:
 * https://docs.gitlab.com/ee/api/notes.html#merge-requests
 * https://docs.gitlab.com/ee/user/project/merge_requests/merge_request_approvals.html

### PR DESCRIPTION
**Pull request by 6543**
close #11137

### CODE REVIEW

File: modules/migrations/gitlab.go [hunk 1]
```diff
@@ -532,5 +533,30 @@ func (g *GitlabDownloader) GetPullRequests(page, perPage int) ([]*base.PullReque
 // GetReviews returns pull requests review
 func (g *GitlabDownloader) GetReviews(pullRequestNumber int64) ([]*base.Review, error) {
 
-	return nil, nil
+	state, _, err := g.client.MergeRequestApprovals.GetApprovalState(g.repoID, int(pullRequestNumber))
+	if err != nil {
+		return nil, err
+	}
+
+	//GitLab only has Approvals witch similar to gitea's approve review's
```
Comment (zeripath) [inline modules/migrations/gitlab.go]:
```suggestion
	// GitLab's Approvals are equivalent to Gitea's approve reviews
```

File: modules/migrations/gitlab.go [hunk 2]
```diff
@@ -532,5 +533,30 @@ func (g *GitlabDownloader) GetPullRequests(page, perPage int) ([]*base.PullReque
...
+			approvers[state.Rules[i].ApprovedBy[u].ID] = state.Rules[i].ApprovedBy[u].Username
+		}
+	}
+
+	var reviews = make([]*base.Review, 0, len(approvers))
+	for id, name := range approvers {
+		reviews = append(reviews, &base.Review{
+			ReviewerID:   int64(id),
+			ReviewerName: name,
+			// GitLab API dont return creation date
```
Comment (zeripath) [inline modules/migrations/gitlab.go]:
```suggestion
			// GitLab API doesn't return a creation date
```

File: modules/migrations/migrate.go [hunk 3]
```diff
@@ -86,7 +86,7 @@ func MigrateRepository(ctx context.Context, doer *models.User, ownerName string,
 	return uploader.repo, nil
 }
 
-// migrateRepository will download informations and upload to Uploader, this is a simple
+// migrateRepository will download information's and upload to Uploader, this is a simple
```
Comment (zeripath) [inline modules/migrations/migrate.go]:
```suggestion
// migrateRepository will download information and then upload it to Uploader, this is a simple
```

**Review by zeripath — APPROVED** [review]

**Review by lafriks — APPROVED** [review]

**Comment by 6543** [comment]
ping lgtm

**Pull request merged**